### DISCUSSION TITLE
How to send request when server initializes in nuxt 3?

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17208] [Questions]
In nuxt 2, i used to use this function in store.
```
export const actions = {
  async nuxtServerInit({commit,app}) {
    await this.$axios.get('/api/auth/csrf')

    await this.$axios.get('/api/auth/seller/info/')
      .then(res => {
        commit('user_set', [true, res.data])
      }) .catch(()=> {
      })
  }
}
```
in nuxt 3, how can I send a request to the backend when the server initializes?

**Comment by Flanker72** [comment]
If you need to send the request ONLY ONCE server was started - you may use the Nitro Plugin approach. To understand how it works you may read my [article about database initialization, Step 2](https://medium.com/@flanker72/nuxt3-complex-solutions-database-integration-8df941f0fb82).
